applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -304,11 +304,18 @@ def get_linear_solver(self):
         return self._linear_solver
 
     def get_builder_and_solver(self):
+        if (self.settings["multi_point_constraints_used"].GetBool() is False and
+            self.GetComputingModelPart().NumberOfMasterSlaveConstraints() > 0):
+            self.settings["multi_point_constraints_used"].SetBool(True)
+            self._builder_and_solver = self._create_builder_and_solver()
         if not hasattr(self, '_builder_and_solver'):
             self._builder_and_solver = self._create_builder_and_solver()
         return self._builder_and_solver
 
     def get_mechanical_solution_strategy(self):
+        if (self.settings["multi_point_constraints_used"].GetBool() is False and
```

Comment:
@loumalouomega why is this needed, it seems to do the exact same

Reply:
source: trust me dude xD

Ok

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/43259040-3b22ad22-90d5-11e8-88a7-ff81c08ab53a.png)

Reply:
This conversation is alive!